behat/symfony2-extension/2.1/Makefile

```diff
@@ -0,0 +1,5 @@
+behat-init:
+	@test -f bin/behat && bin/behat --init || echo "cannot init behat"
+
+behat:
+	@test -f bin/behat && bin/behat || echo "cannot run behat"
```

Comment:
I found that it can be useful now in this way: `@test -f bin/behat && bin/behat --config behat.yaml` because of [this](https://github.com/symfony/recipes-contrib/pull/11/files#r114464402)